### PR TITLE
Add instructions for Linux to the README file

### PR DESCRIPTION
**Pull request by Kr1ss-XD**
**Please review the language** before merging. Not sure if I made it understandable enough :thinking: - so it would be good if a native English speaker checked it out.

_Thx_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,37 @@ duti -s org.dandavison.OpenInEditor file-line-column
...
+GenericName=Open a file at a certain position
+Comment=Opens URLs of the type file-line-column://<path>[:<line>[:<column>]] in the configured editor and positions the cursor
+Icon=text-editor
+Exec=open-in-editor %U
+Categories=Utility;Core;
+StartupNotify=false
+MimeType=x-scheme-handler/file-line-column
+OVER
+
+$ xdg-mime default augmented-open.desktop x-scheme-handler/file-line-column
```
Comment (dandavison) [inline README.md]: Does the second argument need to resolve to the file on disk? I.e. would it be clearer to write this instruction like this?

```
xdg-mime default ~/.local/share/applications/augmented-open.desktop x-scheme-handler/file-line-column
```

**Comment by Kr1ss-XD** [comment]
No, XDG defines a search path for this. `~/.local/share/applications/` is part of it and overrides files of the same name in directories which are earlier in that search path. E.g. `~/.local/share/applications/appX.desktop` has priority over `/usr/share/applications/appX.desktop` if both exist.

File: README.md (hunk 1 above)
Comment (Kr1ss-XD) [reply README.md]: No, XDG defines a search path for this. ~/.local/share/applications/ is part of it and overrides files of the same name in directories which are ~~earlier~~ (E/)later in that search path. E.g. ~/.local/share/applications/appX.desktop has priority over /usr/share/applications/appX.desktop if both exist.

Comment (Kr1ss-XD) [reply README.md]: **PS** This argument repesents the "[desktop file ID](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#desktop-file-id)" (i.e. not a file path).

**EDIT** Fixed the link - it was crippled, sry.

**Comment by dandavison** [comment]
I made a few edits, and changed it so that it's not giving shell commands to write the contents -- is this OK?

My thinking is this is easier to understand for less expert users, and more expert users will know how to create files from the command line.

**Comment by Kr1ss-XD** [comment]
:+1:
That looks fine, thank you very much !

**Comment by dandavison** [comment]
Great, thanks a lot for this @Kr1ss-XD!

**Pull request merged**